Makefile

```diff
@@ -14,12 +14,24 @@ build:
 	@go build -ldflags ${LDFLAGS} ./cmd/celestia
 .PHONY: build
 
-## install: Builds and installs the celestia-node binary into the GOBIN directory.
+## install: Build and install the celestia-node binary into the GOBIN directory.
 install:
 	@echo "--> Installing Celestia"
 	@go install -ldflags ${LDFLAGS}  ./cmd/celestia
 .PHONY: install
 
+## shed: Build cel-shed binary.
+shed:
```

Comment:
```suggestion
build-shed:
```

Reply:
well, this is not Makefile idiomatic tbh

Reply:
so why is install-shed? let's make them consistent please.

Reply:
And the thing I did is similar tbh , it should exactly match the made file name - `cel-shed`

Reply:
actually you're right, let's just leave it make shed.